apps/transport/lib/transport/import_data.ex

```diff
@@ -201,6 +199,7 @@ defmodule Transport.ImportData do
 
   def is_format?(nil, _), do: false
   def is_format?(%{"format" => format}, expected), do: is_format?(format, expected)
+  def is_format?([head | tail], expected), do: is_format?(head, expected) && is_format?(tail, expected)
```

Comment:
How does this work? the `expected` should be the list, right?

at some point, il will call `is_format?(tail, [])` and it won’t match any function.

Maybe we need some unit tests here ;)

Reply:
it was checked by the `is_gtfs?` unit test, but you're right I don't know how it worked :fearful: 

It seems that the tail case is giving an empty string, so the `is_format?("gtfs", "") works because "gtfs" contains ""...